genai-perf/genai_perf/metrics/llm_metrics.py

```diff
@@ -105,4 +109,6 @@ def system_metrics(self) -> List[MetricMetadata]:
         # base metrics first and then task specific metrics. Uncomment the below
         # line to enable this order:
         # return base_metrics + self.LLM_SYSTEM_METRICS
+        # Rightnow the goodput will be printed out before throughput if there is
```

Comment:
This comment seems like it could be removed.

Reply:
Updated